modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/itemselector/plugin.js

```diff
@@ -259,13 +259,19 @@
 
 			if (selectedItem) {
 				const imageSrc = instance._getItemSrc(editor, selectedItem);
+				const editorContent = document.getElementById(
+					editor.id + '_contents'
+				);
+
+				const editorContentHeight = editorContent.getBoundingClientRect()
+					.height;
 
 				if (imageSrc) {
 					if (typeof callback === 'function') {
 						callback(imageSrc, selectedItem);
 					}
 					else {
-						let elementOuterHtml = '<img src="' + imageSrc + '">';
+						let elementOuterHtml = `<img src="${imageSrc}" height="${editorContentHeight}">`;
```

Comment:
Since we only use the `editorContentHeight` within the `else {}` block, we can move `editorContent` and `editorContentHeight` inside that block as well.